S23/kishore223/lowercase.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+
+def lowercase_word(word, lang):
+    if lang in ['tr', 'az']:
+        return word.replace('I', 'ı').lower()
+    elif lang == 'ga':
+        vowels = set(['A', 'E', 'I', 'O', 'U', 'Á', 'É', 'Í', 'Ó', 'Ú'])
+        for i in range(1, len(word)):
+            if word[i-1] in ['n', 't'] and word[i] in vowels and word[i].isupper():
+                return word[:i] + '-' + word[i:].lower()
```

Comment:
This should only be applied at the beginning of a word.    nAthair -> n-athair, but if it's "bhurnAthair, then "bhurnathair" is fine.